### PR TITLE
custom hooks

### PR DESCRIPTION
**Pull request by Rahulsinha197**

### CODE REVIEW

File: redux-mui-crud/src/components/DocTitleOne.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { useState } from "react";
+import useDocumentTitle from "../hooks/useDocumentTitle";
+
+function DocTitleOne() {
+  const [count, setCount] = useState(0);
+
+  useDocumentTitle(count);
+  return (
+    <div>
```
Comment (simformniki) [inline redux-mui-crud/src/components/DocTitleOne.js]: No need to wrap with div.  you can directly return button.

File: redux-mui-crud/src/pages/VisitCustom.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import DocTitleOne from "../components/DocTitleOne";
+import DocTitleTwo from "../components/DocTitleTwo";
+
+function VisitCustom() {
+  return (
+    <div>
```
Comment (simformniki) [inline redux-mui-crud/src/pages/VisitCustom.js]: you can use fragment to wrap 2 components.

**Comment by simformniki** [comment]
Can you please explain me on Monday what did you understand in this implementation.

**Comment by Rahulsinha197** [comment]
Yes sure....

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**